vignettes/reactlog.Rmd

```diff
@@ -0,0 +1,346 @@
+
+
+#### Reactive Graph Nodes
+
+```{r}
+knitr::include_graphics("images/elements.png")
+```
+
+Reactive inputs are values that are the beginning of a reactive chain.  They are the leftmost starting points of the reactive graph.  Reactive sources are typically values created in `input`, `clientData`, `shiny::reactiveVal`, or `shiny::reactiveValues`.
+
+Reactive expressions are conductors between sources and enpoints.  Their values depend on other reactive elements and are depended on by other reactive elements.  These expressions help with intermediate calculations, preventing duplicate calculations, and aid in logical flow.
+
+Reactive endpoints are values that are being stored to the `output` object or are `observe`ing other reactive elements to produce side effects.
+
+> Pro tip: Use the `label` parameter when defining reactive elements.  This will help your debug process and avoid long, code-exposing reactive element labels.
+
+## Actions
+
+#### Mouse
+
+* Navigation Buttons
+  * **Click**: Move to the next (or previous) appropriate step.
+* Progress Bar:
+  * **Click (+drag)**: Move to the step in the progress bar immediately.  This still works even if the targeted step is not in a filtered reactive graph.
+* Reactive Element
+  * **Hover**: Highlight the particular element's family tree within the graph.
+  * **Click**: Maintain the highlighting of a particular element's family tree even after the mouse leaves the element.
+  * **Double Click**: Subset the graph displayed to the family tree of the element clicked.
+
+#### Keyboard Shortcuts
+
+* Navigation
+  * `Left/Right Arrow keys`: Left / Right a single Step.
+  * `Left/Right Arrow keys + OPTION`: Move to the nearest Reactive Endpoint Calculation.
+  * `Left/Right Arrow keys + OPTION + SHIFT`: Move to the nearest Idle Step.
+  * `Home/End`: Move to the nearest User Marked Step or the beginning or end of the reactlog.
+
+> Pro tip: Hold down the arrow keys to quickly iterate over all the steps of the displayed graph.
+
+* `Esc`
+  * If an element has been clicked, remove the click selection.
+  * If the cursor focus is on the search bar, move the focus to the reactive graph.
+  * If the focus is on the graph and the graph is filtered, reset the filter and display the whole graph.
+* Reactlog Info
+  * `s`: Move the focus to the **s**earch bar
+  * `l`: When the cursor focus is on the graph, display the react**l**og step information.
+  * `f`: Resize the graph to **f**it on the screen
+
+## Reactive States
+
+```{r, out.width="15%"}
+knitr::include_graphics("images/legend.png")
+```
```

Comment:
Image looks to be out-of-date

Reply:
This is the correct image.  I have updated the order within this branch.